### PR TITLE
Support windows

### PR DESCRIPTION
**Pull request by nllong**
Start of a pull request for supporting windows (and mac)

### CODE REVIEW

File: lib/openstudio/workflow/jobs/run_energyplus/run_energyplus.rb [hunk 1]
```diff
@@ -117,6 +112,45 @@ def perform
 
   private
 
+  # Look for the location of EnergyPlus
+  def find_energyplus
+    if ENV['ENERGYPLUSDIR']
+      return ENV['ENERGYPLUSDIR']
+    elsif ENV['RUBYLIB'] =~ /OpenStudio/
```
Comment (nllong) [inline lib/openstudio/workflow/jobs/run_energyplus/run_energyplus.rb]: @macumber, I am not happy about this method and it really isn't finished yet. Can we chat about this? My thinking is that if we are shipping energyplus with openstudio, then let's just use that one. If so, then is there a method in openstudio to get the energyplus fqp?

Comment (macumber) [inline lib/openstudio/workflow/jobs/run_energyplus/run_energyplus.rb]: I don't think we currently have this method in OpenStudio but I think that we should add it here: https://github.com/NREL/OpenStudio/blob/develop/openstudiocore/src/utilities/core/ApplicationPathHelpers.hpp

@kbenne is that what you are thinking too?

File: lib/openstudio/workflow/jobs/run_energyplus/run_energyplus.rb [hunk 2]
```diff
@@ -117,6 +112,45 @@ def perform
...
+  end
+
+  def prepare_energyplus_dir
+    def copy_if_exists(from, to_dir)
+      @logger.info "Copying #{from} to #{to_dir}"
+      FileUtils.copy(from, "#{to_dir}/#{File.basename(from)}") if File.exists?(from)
+    end
+
+    @logger.info "Copying EnergyPlus files to run directory: #{@run_directory}"
+    copy_if_exists("#{@options[:energyplus_path]}/libbcvtb.so", @run_directory)
```
Comment (macumber) [inline lib/openstudio/workflow/jobs/run_energyplus/run_energyplus.rb]: @nllong maybe we want to look into the new command line interface options for E+ 8.3, might be able to get rid of some of this crap

**Comment by nllong** [comment]
@macumber As far as I can tell, I've ripped out all Mac/Linux based dependencies. I am having a little trouble testing on Windows, so maybe you take a crack at this. You would need to check this out and run (probably from a git bash):

```
bundle update
bundle exec rake
```

There is one test case that fails when the suite is run, but passes if ran independently.

**Pull request merged**